src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -438,6 +442,21 @@ def _handleNoStatusUpdate(self, workload, request_args, dn):
                 msg = "There were unhandled arguments left for no-status update."
                 raise InvalidSpecParameterValue(msg) from ex
 
+        try:
+            # make HTTP POST request to /ms-transferor/data with {'workflow': <workflow>} payload
+            # but only if it is necessary, i.e. the workqueu_stat_validation checks
+            # that only supported parameter change. It would be either RequestPriority or SiteXXXList
+            # and we also check reqStatus has a right value when site lists change is required
+            if reqStatus != 'assignment-approved':
+                # the makeHttpRequest function will call MSTransferor because site list change occured
+                rurl = f"{self.msTransferorUrl}/data"
```

Comment:
Don't we have to update this endpoint? Even if it works with this endpoint as well, I would suggest to use the "full endpoint", example: `/ms-transferor/data/transferor`.